### DISCUSSION TITLE
Show current search keywords when click the textbox in the search engine settings.

### DISCUSSION THREAD

**Discussion by MoltenMonster** [Floorp-Projects/Floorp#1010] [Feature Request]
This is also an issue with Firefox. You can't easily change / edit keywords, they just get replaced.
<img width="693" alt="Screenshot 2024-03-16 at 10 53 41 PM" src="https://github.com/Floorp-Projects/Floorp/assets/42647030/4fb83319-0053-457f-910b-ab067102bf59">
<img width="708" alt="Screenshot 2024-03-16 at 10 53 48 PM" src="https://github.com/Floorp-Projects/Floorp/assets/42647030/f5c1632b-3a54-4841-a3da-b6ceb24b81a2">

**Comment by CutterKnife** [comment]
Default keywords will not appear, but keywords you add yourself will appear there. I believe this is a Firefox specification.
